src/Commands/AddCommand.php

```diff
@@ -38,11 +38,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $gitDir = $input->getOption('git-dir');
         $force = $input->getOption('force');
         $forceWindows = $input->getOption('force-win');
-        $hookDir = "{$gitDir}/hooks";
 
-        if (! is_dir($hookDir)) {
-            mkdir($hookDir, 0700, true);
-        }
+        create_hooks_dir("{$gitDir}/hooks");
```

Comment:
last nitpick: I would prefer `create_hooks_dir($gitDir);` and then the string interpolation will happen in the function. that removes some redundancy. what do you think

Reply:
Yes, much better! I did not pay attention to this before :sweat_smile:, I'll make the change.